controller/tests/array_action/svc/array_mediator_svc_test.py

```diff
@@ -888,3 +892,57 @@ def test_get_array_fc_wwns_success(self):
         self.svc.client.svcinfo.lsfabric.return_value = [port_1, port_2]
         wwns = self.svc.get_array_fc_wwns('host')
         self.assertEqual(wwns, ['5005076810282CD8', '5005076810262CD8'])
+
+    def _prepare_mocks_for_expand_volume(self):
+        volume = Mock(as_single_element=Munch({'vdisk_UID': 'vol_id',
+                                               'name': 'test_vol',
+                                               'capacity': '512',
+                                               'mdisk_grp_name': 'pool_name'
+                                               }))
+        self.svc.client.svcinfo.lsvdisk.return_value = volume
+        self.svc.client.svcinfo.lsfcmap.return_value = Mock(as_list=[])
+
+    def test_expand_volume_success(self):
+        self._prepare_mocks_for_expand_volume()
+        self.svc.expand_volume('vol_id', 1024)
```

Comment:
consider making it/a separate test more interesting with `513` instead of `1024`

Reply:
done